cfme/automate/provisioning_dialogs.py

```diff
@@ -180,11 +159,9 @@ def update(self, updates, cancel=False, reset=False):
             flash_msg = ('Dialog "{}" was saved'.format(updates.get('description') or
                                                         self.description))
             btn = view.form.save
-
+            view = self.create_view(ProvDiagDetailsView, override=updates)
```

Comment:
**Required**

Two test failures in PRT from this - override should only be passed if `cancel == False`, otherwise the is_displayed check fails because the updates were not saved.

Reply:
Fixed.